src/test/java/com/antkorwin/betterstrings/BetterStringsProcessorTest.java

```diff
@@ -83,6 +83,19 @@ void createNewClassInExpression() {
 		assertThat(result).isEqualTo("password = 1234");
 	}
 
+	@Test
+	void interpolationInFields() {
```

Comment:
Actually, if it comes to this entire test suite... to make sure all stuff works both for `-AcallToStringExplicitlyInInterpolations` and without while avoiding test code duplication, you could use https://github.com/junit-team/junit4/wiki/Parameterized-tests (although most likely Maven setup for JUnit would need to be adjusted first - hence I haven't done it myself on PR #14)

Reply:
I thought about parameterized tests, but It may increase the complexity of the tests code base. Each test case will require one additional argument with another compiler flags and will require to use annotation with the source of all parameters.

There is another approach - we can run all tests with this flag by the use of another build script. The same code base of tests, but different build script commands. One job without compiler argument and the second turned it on. But in this case, if we need to test behavior which specifical for one compiler flag then we need to exclude manually this test from build job.

I want to try and compare these approaches.